docker/versions.dockerfile

```diff
@@ -14,11 +14,9 @@ ARG HASKELL_STACK_VERSION=2.13.1
 ARG JAVA_VERSION=21
 ARG LICENSEE_VERSION=9.18.0
 ARG NODEJS_VERSION=22.13.0
-ARG NPM_VERSION=10.9.2
```

Comment:
Does this imply and new requirements on the lowest versions of the individual tools (i.e., do the tools themselves also need to support corepack in some way), which should be related in `getVersionRequirement()` of the tools?

Reply:
No, AFAIK corepack just installs the tools on demand but they do not have to be aware of corepack.